### DISCUSSION TITLE
Modified target increments release to 6.0.0, configurator does not work

### DISCUSSION THREAD

**Discussion by clarkpark** [iNavFlight/inav#8208] [Q&A] [answered]
I made a minor change to my target FC, compiled, and the release is incremented to 6.0.0.  So the 5.0 configurator doesn't work.  Is there a way to fix this from either the compile side or the configurator side?  Sorry if this is a stupid question, I am out of my element here.
fyi, here is link to discussion on the change: https://github.com/iNavFlight/inav/discussions/8193

**Comment by DzikuVx** [comment, marked as answer]
Build Configurator locally from master. Everything is in the readme of the Configurator project

**Comment by stronnag** [comment]
Development snapshot 6.0 configurator

http://seyrsnys.myzen.co.uk/inav-configurator-next/
